cyclops/monitor/clinical_applicator.py

```diff
@@ -34,7 +34,13 @@ def __init__(self, shift_type: str, source, target, shift_id: str = None):
         if self.shift_type not in self.shift_types:
             raise ValueError(f"Shift type {self.shift_type} not supported. ")
 
-    def apply_shift(self, dataset: Dataset):
+    def apply_shift(
+        self,
+        dataset: Dataset,
+        batched: bool = True,
+        batch_size: int = 1000,
+        num_proc: int = 1,
+    ):
         """apply_shift.
```

Comment:
docstring could be improved with more info of what the method does and also `Parameters` info is missing